plugins/org.orbisgis.h2gis/bin/plugin.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+
+<plugin>
+
+<extension point="org.jkiss.dbeaver.dataSourceProvider">
+
+        <!-- H2GIS driver -->
+        <datasource
+                class="org.orbisgis.h2gis.H2GIS200DataSourceProvider"
+                description="%datasource.h2gis.description"
+                id="h2gis_2.0.0"
+                parent="generic"
+                label="H2GIS"
+                icon="icons/h2gis_icon.png"
+                dialect="h2">
+            <drivers managable="true">
+
+                <driver
+                    id="h2gis_embedded_2_0_0"
+                    label="H2GIS Embedded 2.0.0"
+                    icon="icons/h2gis_icon.png"
+                    iconBig="icons/h2gis_icon_big.png"
+                    category="H2"
+                    class="org.h2.Driver"
+                    sampleURL="jdbc:h2:{file}"
+                    webURL="http://www.h2gis.org/"
+                    embedded="true"
+                    description="H2GIS Embedded"
+                    categories="spatial,sql,embedded">
+
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis/2.0.0-SNAPSHOT/h2gis-2.0.0-20220110.125034-16.jar"/>
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-api/2.0.0-SNAPSHOT/h2gis-api-2.0.0-20220110.125034-16.jar"/>
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-network/2.0.0-SNAPSHOT/h2gis-network-2.0.0-20220110.125034-16.jar"/>
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-osgi/2.0.0-SNAPSHOT/h2gis-osgi-2.0.0-20220110.125034-16.jar"/>
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-test-utilities/2.0.0-SNAPSHOT/h2gis-test-utilities-2.0.0-20220110.125034-16.jar"/>
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-utilities/2.0.0-SNAPSHOT/h2gis-utilities-2.0.0-20220110.125034-16.jar"/>
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/postgis-jts/2.0.0-SNAPSHOT/postgis-jts-2.0.0-20220110.125034-16.jar"/>
```

Comment:
This PR must be updated with the current master because you have here the old jar files.